src/main/resources/frontend/src/views/PhaseView/RubricTableRow.vue

```diff
@@ -0,0 +1,51 @@
+<script setup lang="ts">
+import type {RubricItem, TestResult} from "@/types/types";
+
+defineProps<{
+  rubricItem: RubricItem;
+  resultsClicked: () => void;
+}>();
+
+const failedTests = (testResults: TestResult) => {
+  return testResults && (testResults.numTestsFailed > 0 || testResults.numTestsPassed === 0);
+}
+
+const addColor = (qualityNotes: string) => {
+  return qualityNotes.replace(/✓/g, `<span class="success">✓</span>`).replace(/✗/g, `<span class="failure">✗</span>`);
```

Comment:
Note: Javascript offers an [String.replaceAll()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replaceAll) method which may be helpful here.

Reply:
IntelliJ gives an error when using replaceAll, which is why I opted for just using replace. Paul said he'd rather have old javascript than have errors

Reply:
`replaceAll` was added with ES2021 (https://github.com/daumann/ECMAScript-new-features-list/blob/main/ES2021.MD), and for some reason our TypeScript configuration is running against an older specification.

I looked into this before, but couldn't get TypeScript to run against the newer version. I'm sure it's possible, so if someone knows how to fix that I'd be all for it.

Reply:
Here is the [Vue documentation](https://v2.vuejs.org/v2/guide/typescript) on using Typescript.

It looks like the current app hasn't had that configuration, but I'll propose some changes to see if they solve the problem

Reply:
This issue should be resolved with #223. After merging that PR into this branch, we should be allowed to use `.replaceAll()` without complaints.